### DISCUSSION TITLE
500 (Looks like something went wrong!)

### DISCUSSION THREAD

**Discussion by jb3008** [community/community#33137] [Other Feature Feedback, Questions, & Ideas] [answered]
https://github.com/ showing 500 .

**Comment by radar-blip** [comment, marked as answer]
Same thing happening to several people right now. Signing into your profile page seems to work fine.

**Comment by jb3008** [comment]
Ok thanks for your quick response

On Thu, 15 Sep 2022 at 11:15 AM, David Lawson ***@***.***>
wrote:

> Same thing happening to several people right now. Signing into your
> profile page seems to work fine.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/33137#discussioncomment-3650558>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANND7ZWU2FGIWZS36Q6QL6DV6KZYBANCNFSM6AAAAAAQNA77A4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Thanks & Regards

Jayesh Bhanderi.
*Mob No: 9727190190*

**Reply by radar-blip** [reply]
No problem. Good luck!

**Comment by versona-tech** [comment]
f ** github It does not contain support only a handful of idiots
